README.md

```diff
@@ -65,10 +65,10 @@ rustup install nightly
 * OpenCL bindings
 
 ```shell
-# Ubuntu
+# Ubuntu (20.04)
 sudo make install-deps
 # Or
-sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
+sudo apt install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev cmake
```

Comment:
This doesn't belong here: `-no-install-recommends -y`

Reply:
Fixed